test/Microsoft.ML.Tests/OnnxConversionTest.cs

```diff
@@ -403,6 +404,127 @@ public void MulticlassLogisticRegressionOnnxConversionTest()
+                    'output_model': '{1}'
+                }}
+            }}", dataPath.Replace("\\", "\\\\"), modelPath.Replace("\\", "\\\\"));
+
+            // Write entry point graph into file so that it can be invoke by graph runner below.
+            var jsonPath = DeleteOutputPath("graph.json");
+            File.WriteAllLines(jsonPath, new[] { inputGraph });
+
+            // Execute the saved entry point graph to produce a predictive model.
+            var args = new ExecuteGraphCommand.Arguments() { GraphPath = jsonPath };
+            var cmd = new ExecuteGraphCommand(Env, args);
+            cmd.Run();
+
+            // Make entry point graph to conduct ONNX conversion.
+            inputGraph = string.Format(@"
+            {{
+                'Inputs': {{
+                    'model': '{0}'
+                }},
+                'Nodes': [
+                    {{
+                        'Inputs': {{
+                            'Domain': 'com.microsoft.models',
+                            'Json': '{1}',
+                            'PredictiveModel': '$model',
+                            'Onnx': '{2}',
+                            'OnnxVersion': 'Experimental'
+                        }},
+                        'Name': 'Models.OnnxConverter',
+                        'Outputs': {{}}
+                    }}
+                ],
+                'Outputs': {{}}
+            }}
+            ", modelPath.Replace("\\", "\\\\"), onnxJsonPath.Replace("\\", "\\\\"), onnxPath.Replace("\\", "\\\\"));
+
+            // Write entry point graph for ONNX conversion into file so that it can be invoke by graph runner below.
+            jsonPath = DeleteOutputPath("graph.json");
+            File.WriteAllLines(jsonPath, new[] { inputGraph });
+
+            // Onnx converter's assembly is not loaded by default, so we need to register it before calling it.
+            Env.ComponentCatalog.RegisterAssembly(typeof(OnnxExportExtensions).Assembly);
+
+            // Execute the saved entry point graph to convert the saved model to ONNX format.
+            args = new ExecuteGraphCommand.Arguments() { GraphPath = jsonPath };
+            cmd = new ExecuteGraphCommand(Env, args);
+            cmd.Run();
+
+            // Load the resulted ONNX model from the file so that we can check if the conversion looks good.
+            var model = new OnnxCSharpToProtoWrapper.ModelProto();
+            using (var modelStream = File.OpenRead(onnxPath))
+                model = OnnxCSharpToProtoWrapper.ModelProto.Parser.ParseFrom(modelStream);
+
+            // Make sure a PredictorModel is loaded by seeing if a predictive model exists. In this the
+            // predictive model is "LinearRegressor" (converted from StochasticDualCoordinateAscentRegressor
+            // in the original training entry-point graph.
+            Assert.Equal("Scaler", model.Graph.Node[0].OpType);
+            Assert.Equal("LinearRegressor", model.Graph.Node[1].OpType);
```

Comment:
> Assert.Equal("LinearRegressor", model.Graph.Node[1].OpType); [](start = 11, length = 61)

Is it possible to run original model and converted models and see if results are the same?